install/templates/eth2.tmpl

```diff
@@ -8,54 +8,57 @@
 version: "3.7"
 services:
   eth2:
-    image: ${BN_CONTAINER_TAG}
+    image: {{.GetBeaconContainerTag}}
     user: root
-    container_name: ${COMPOSE_PROJECT_NAME}_eth2
+    container_name: {{.Smartnode.ProjectName.Value}}_eth2
     restart: unless-stopped
     stop_grace_period: 3m
-    ports: [ "${BN_P2P_PORT:-9001}:${BN_P2P_PORT:-9001}/tcp", "${BN_P2P_PORT:-9001}:${BN_P2P_PORT:-9001}/udp"${BN_OPEN_PORTS}, "${BN_P2P_QUIC_PORT:-8001}:${BN_P2P_QUIC_PORT:-8001}/udp"]
+    {{- $p2p := (or .ConsensusCommon.P2pPort.Value "9001")}}
```

Comment:
It's not clear where the quic port is now.

Reply:
Looks like i missed it